include/zephyr/shell/shell.h

```diff
@@ -886,10 +891,54 @@ struct shell_ctx {
 
 	struct k_poll_signal signals[SHELL_SIGNALS];
 
+#if defined(CONFIG_SHELL_BYPASS_USER)
+	/** The ID of the user bypass thread.
+	 */
+	k_tid_t bypass_user_thread;
+
+	/** This is the number of received bytes dropped during the current or
+	 * most recent user bypass operation. It is reset to 0 on a syscall to
+	 * shell_bypass_enter().
```

Comment:
not in keeping with existing comments, see https://github.com/zephyrproject-rtos/zephyr/blob/12ecee08eedb93b75a138137050d13fd570b4e4d/include/zephyr/shell/shell.h#L85